### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by jmcarp**
* Get selenium tests running again on travis / sauce labs
* Pull in various updates from cfpb/regulations-site (copied instead of cherry-picking due to conflicts)
* Fix a few brittle tests

Notes:
* This is using a personal sauce labs account until I get a real account from devops. Submitting now in case anybody wants to review.
* The tests are still pretty brittle! It should also be easier to run them locally without going through sauce labs.
* Builds would be faster if we ran against different browsers in parallel.
* Hopefully the build for this PR passes on travis, but that doesn't mean that this actually works, since we don't actually install sauce connect or run tests for pull requests. There are some passing builds here: https://travis-ci.org/jmcarp/regulations-site/builds.

[Resolves https://github.com/18F/atf-eregs/issues/306]

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -130,10 +130,6 @@ module.exports = function(grunt) {
      },
 
     shell: {
-      'build-require': {
-        command: './require.sh'
-      },
-
```
Comment (cmc333333) [inline Gruntfile.js]: I have no idea what this was for. What did it do that we don't want anymore?

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     }
   ],
   "engines": {
-    "node": ">=0.12.0"
+    "node": "4.4.0"
```
Comment (cmc333333) [inline package.json]: :sweat_smile:

**Comment by cmc333333** [comment]
Looks good -- I'd prefer swapping out the credentials before merging, if that seems like a quick thing.

**Comment by cmc333333** [comment]
Also :clap: for using css selectors over xpath.

File: Gruntfile.js (hunk 1 above)
Comment (jmcarp) [inline Gruntfile.js]: Looks like it ran a script that was deleted about a year ago when the repo dropped requirejs--see https://github.com/18F/regulations-site/commit/3ad9df710583f5d694aa3f0118ca41dd00fcba61. There aren't any references to this grunt task left in the project (which is good!), so deleting should be safe.

**Comment by jmcarp** [comment]
I'll update the sauce credentials after https://github.com/18F/Infrastructure/issues/582 is resolved.

**Comment by cmc333333** [comment]
We should have access to credentials now. Not sure where the branch conflict is

**Comment by jmcarp** [comment]
Conflicts resolved. I also rebuilt the sauce labs credentials to target this repo--although they wound up being the same credentials, since my previously personal account is now a sub-account of the 18F account. I guess that's a thing.

**Comment by cmc333333** [comment]
Nice work

**Pull request merged**